pages/index.js

```diff
@@ -9,11 +9,10 @@ const Countdown = dynamic(() => import('react-countdown'), {
 })
 
 const deadlines = [
-  { milestone: "End of development", date: "2022-11-08", colors: "bg-slate-800 text-slate-100 hover:text-amber-100 hover:bg-amber-600", c: "bg-amber-600/50" },
+  { milestone: "End of development", date: "2022-12-06", colors: "bg-slate-800 text-slate-100 hover:text-amber-100 hover:bg-amber-600", c: "bg-amber-600/50" },
   { milestone: "Lisbon events kickoff", date: "2022-10-24", colors: "bg-slate-800 text-slate-100 hover:text-pink-100 hover:bg-pink-600", c: "bg-pink-600/50" },
-  { milestone: "Audits kickoff", date: "2022-11-14", colors: "bg-slate-800 text-slate-100 hover:text-red-100 hover:bg-red-600", c: "bg-red-600/50" },
-  { milestone: "Butterfly launch", date: "2022-11-14", colors: "bg-slate-800 text-slate-100 hover:text-violet-100 hover:bg-violet-600", c: "bg-violet-600/50" },
-  { milestone: "Buildernet launch", date: "2022-11-28", colors: "bg-slate-800 text-slate-100 hover:text-sky-100 hover:bg-sky-600", c: "bg-sky-600/50" },
+  { milestone: "Audits kickoff", date: "2022-11-22", colors: "bg-slate-800 text-slate-100 hover:text-red-100 hover:bg-red-600", c: "bg-red-600/50" },
+  { milestone: "X testnet launch (former Buildernet, name pending)", date: "2022-12-06", colors: "bg-slate-800 text-slate-100 hover:text-sky-100 hover:bg-sky-600", c: "bg-sky-600/50" },
```

Comment:
This long name breaks the layout.

```suggestion
  { milestone: "Stable testnet + Acceleration", date: "2022-12-06", colors: "bg-slate-800 text-slate-100 hover:text-sky-100 hover:bg-sky-600", c: "bg-sky-600/50" },
```